### PR TITLE
Set up for Minecraft 1.21.1

### PR DESCRIPTION
**Pull request by nicdgonzalez**
I updated the properties in `gradle.properties` to work for Minecraft 1.21.1 as per [Develop | Fabric](https://fabricmc.net/develop/).

### CODE REVIEW

**Comment by 48kMcFae** [comment]
what does the process look like for getting this fixed for 1.21.1? in layman terms lol, or is it still scuffed

**Comment by nicdgonzalez** [comment]
> what does the process look like for getting this fixed for 1.21.1? in layman terms lol, or is it still scuffed

I was able to compile the mod, and load it into Minecraft 1.21.1, but the resource pack that goes along with it doesn't work. I was hoping to hear back by now, but since I haven't I may just try to recreate it myself (though I don't know how long that takes or what that entails).

If you're interested, I will come back and update you after I get it working 👍🏽

**Comment by keve1227** [comment]
I'm sorry for not maintaining this more regularly. I've been preoccupied with work these past few months. Right now I'm in the process of updating the mod and the resource pack as well.

Thank you for contributing 😁

**Comment by nicdgonzalez** [comment]
> I'm sorry for not maintaining this more regularly. I've been preoccupied with work these past few months. Right now I'm in the process of updating the mod and the resource pack as well.
> 
> Thank you for contributing 😁

Nooooo, don't sweat it! We appreciate the fact that you made this open source in the first place. You are the one doing us a favor here.

I'd be happy to help you out with this in the future! I'll study up on the codebase so you don't have to tackle it alone next time. Thank you for all of your hard work! お疲れさまです！

**Comment by 48kMcFae** [comment]
so proud of this community
